### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by harizfajar** [leokhoa/laragon#809] [General]
how to fix
![image](https://github.com/leokhoa/laragon/assets/129683654/b4b2b2b3-1bcd-4966-8e15-b6f958ee9981)

**Comment by rescenic** [comment]
Check if auto virtual host is on. Close the browser. 
![laragon](https://github.com/leokhoa/laragon/assets/9632756/6f969c36-1d9f-4c3e-8224-7df3011b5c4b)
And then Add laragon.crt to Trust Store.
![laragon crt](https://github.com/leokhoa/laragon/assets/9632756/6504b91a-1269-4ba9-bd3c-e85d4de89e6e)
Open again the browser, check again laravel11.test
